proc/threads.c

```diff
@@ -1357,7 +1368,10 @@ int threads_sigpost(process_t *process, thread_t *thread, int sig)
 			proc_kill(process);
 			break;
```

Comment:
While you're here and have removed `/* passthrough */` comment - could you pls replace invalid `break;` here with `return EOK`?